server/api/src/rsDataObjOpen.cpp

```diff
@@ -212,6 +212,9 @@ namespace
             // and the operation is supposed to turn into an overwrite of the existing data object.
             // Return a value of 0 here and the caller will continue with the open operation.
             if (!creating_new_replica) {
+                irods::log(LOG_DEBUG, fmt::format(
+                    "[{}:{}] - switching to overwrite an existing replica [path=[{}], hierarchy=[{}]",
```

Comment:
add missing second closing bracket at end of hierarchy

Reply:
Done